### PR TITLE
Add RukaMan Coordinates

### PR DESCRIPTION
**Pull request by RukaMan**
@githubteacher 

Seattle, WA

### CODE REVIEW

**Comment by bryndwl** [comment]
awesome 👍

**Comment by klavavej** [comment]
nice PR 🎉

File: _pins/Anjoal.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Anjoal
+latitude: 36.527061
```
Comment (klavavej) [inline _pins/Anjoal.json]: such latitude

**Comment by beardofedu** [comment]
@RukaMan Closing this issue, you are merging Master into your branch, when you recreate the pull request, select the base of `master` and the `compare` of your branch.

**Pull request closed without merging**